torch/_decomp/__init__.py

```diff
@@ -328,6 +328,7 @@ def core_aten_decompositions() -> Dict[torch._ops.OperatorBase, Callable]:
             aten.leaky_relu_backward,
             aten.lerp,
             aten.lerp_,
+            aten.lift_fresh,
```

Comment:
The lift_fresh decomposition is just a noop, and it has a note saying that it's not correct under autograd. Do we want to add it as a core decomp?

Reply:
My understanding is that the core decompositions are registered to the `"post_autograd"` decomposition table (see `torch/_decomp/__init__.py`), presumably that means that these decompositions should be used when autograd is not a factor.

This is a good point though, would like to get additional confirmation that what I said above makes sense.

cc: @SherlockNoMad @bdhirsh @jansel

Reply:
hmm @ezyang - today when we see a tensor constant during tracing, that turns into a `constant_lifted = aten.lift_fresh.default(self._tensor_constant0)` in the graph. Do you think `lift_fresh` should be a first-class concept for backends?

Reply:
we guarantee that a graph of core ATen IR is always functional (right @SherlockNoMad?). Then it sounds reasonable for core aten to no-op lift_fresh.